substrate/frame/support/src/transaction_extensions.rs

```diff
@@ -0,0 +1,89 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Transaction extensions.
```

Comment:
Main question that arises (and the doc comment is not particularly useful about it) is whether tx extensions are a frame concept or a non-frame concept? 

The answer is probably latter, but then this file should say what it is, and it is providing. 

It seems to me that `VerifyMultiSignature` could live outside of FRAME actually.

Reply:
> the doc comment is not particularly useful about it

:joy: 

It's not clear if it's a FRAME concept or not, but in my understanding it's tied to the account model that FRAME currently provides. I wouldn't mind leaving it here, but maybe that doc comment would benefit from a revamp.